src/wccp2.cc

```diff
@@ -2024,10 +2033,16 @@ wccp2AssignBuckets(void *)
                                     tmp_rtr,
                                     &wccp_packet,
                                     offset);
+                    // XXX: setup fd_table[] properly so we can use FD_WRITE_METHOD()
+                    fd_table[theWccp2Connection->fd].bytesWritten(offset);
```

Comment:
Same here: Missing protection from comm_udp_sendto() failures. GitHub will not let me post a specific correction for this one.